README.md

```diff
@@ -8,6 +8,25 @@ If you are a Swiss Alpine Club section, be sure to get in touch with Bern in ord
 secret.
 
 
+## Default FE Loginbox
+
+This extension integrations with the felogin systemextension and provides a new marker ###OPENID_CONNECT### to be used 
+in the felogin template. A sample template is included. Once the users clicks the new openid connect link, he/she will 
```

Comment:
- the users clicks => the user clicks
- he/she => she (I figured out by reading many presentations/texts from native speakers that it's very common to use the feminine form when speaking about a user)